From 00e5104b8b0160100bcd442dfcdba507a3d4a137 Mon Sep 17 00:00:00 2001
From: Chuck Short <chuck.short@canonical.com>
Date: Thu, 26 Apr 2012 10:26:57 -0400
Subject: [PATCH] Fix call to network_get_all_by_uuids().

Fix bug 986922

This patch fixes a typo in the network manager.  This function is part
of the db API, not a method of the network manager.

(cherry picked from commit cc05a8defbb0d36ae6531acd2c5a9677c2f7ce97)

Change-Id: I2b0769f69ec4f58cd22708070b5fa4037f856026
---
 nova/network/manager.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/nova/network/manager.py b/nova/network/manager.py
index 9ecc6e2..73ac52b 100644
--- a/nova/network/manager.py
+++ b/nova/network/manager.py
@@ -868,7 +868,7 @@ class NetworkManager(manager.SchedulerDependentManager):
         #                 a non-vlan instance should connect to
         if requested_networks is not None and len(requested_networks) != 0:
             network_uuids = [uuid for (uuid, fixed_ip) in requested_networks]
-            networks = self.network_get_all_by_uuids(context, network_uuids)
+            networks = self.db.network_get_all_by_uuids(context, network_uuids)
         else:
             try:
                 networks = self.db.network_get_all(context)
